### PR TITLE
Add real FAQs

### PR DESCRIPTION
**Pull request by carise**

### CODE REVIEW

File: _faq/general/create-issue.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: I have an idea/found a bug. What do I do next?
+order: 2
+---
+{::options auto_ids="false" /}
```
Comment (mirons-google) [inline _faq/general/create-issue.md]: Help me understand why we're disabling these?

File: _faq/installation_and_deployment/change-setup.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: I had a typo in my Forseti setup, how do I update this?
```
Comment (mirons-google) [inline _faq/installation_and_deployment/change-setup.md]: nit: "I made a mistake in my Forseti setup,..."

File: _faq/installation_and_deployment/separate-project.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Why run Forseti in its own separate project?
+order: 4
+---
+{::options auto_ids="false" /}
+
+Setting Forseti up in a separate project ensures that Forseti has the necessary 
+quota for API calls and managing service accounts and roles. This also 
+restricts who has access to your Forseti-related resources.
```
Comment (mirons-google) [inline _faq/installation_and_deployment/separate-project.md]: Is there a relevant section where we can point the user to in the best practices document that explains our rationale for this?

File: _faq/security_implications/gsuite-key.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+title: Why does Forseti store the G Suite private key in the Compute Engine instnce?
+order: 1
+---
+{::options auto_ids="false" /}
+
+The
```
Comment (mirons-google) [inline _faq/security_implications/gsuite-key.md]: This doesn't really answer the question per-say. The real reason is the API requires certain oauth methods and the oauth library we use requires the key to be local to the code.

File: _faq/security_implications/read-data.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+{::options auto_ids="false" /}
+
+Forseti uses a service account which is granted roles on the organization 
+IAM policy. Because roles are hierarchical in GCP, any role that someone has 
+on the organization level IAM will also be inherited on a lower level, such as 
+at the folder or project level. For example, if you grant the "Browser" role 
+to someone on the organization, they will also be able to see folders and 
+projects within the organization.
+
+For more information, please refer to ["Service account for Forseti Security"]({% link _docs/guides/best-practices.md %}#service-account-for-forseti-security).
```
Comment (mirons-google) [inline _faq/security_implications/read-data.md]: nit break before the hyper-link.

File: _faq/general/create-issue.md (hunk 1 above)
Comment (carise) [reply _faq/general/create-issue.md]: No idea what those are.

File: _data/faq_categories.yml [hunk 6]
```diff
@@ -1,4 +1,6 @@
 # The order of the categories in this file (first-to-last) determines their
 # order in the UI (top-to-bottom)
-- title: Category 1
-- title: Category 2
+- title: General
```
Comment (Tabrarian) [inline _data/faq_categories.yml]: Headings should generally be sentence case: https://developers.google.com/style/headings So, for example, "Installation and deployment".

File: _faq/installation_and_deployment/org-data-access.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+---
+title: Why is Forseti only reading data from one project?
+order: 5
+---
+{::options auto_ids="false" /}
+
+If Forseti is reading data from only one project, your Forseti service account 
+might have access only to that particular project. To get read access to all of 
+the projects under your organization, add the service account to the
+organization IAM policy with the
```
Comment (Tabrarian) [inline _faq/installation_and_deployment/org-data-access.md]: "Cloud IAM" when referring to the product :)

File: _faq/installation_and_deployment/org-data-access.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+order: 5
+---
+{::options auto_ids="false" /}
+
+If Forseti is reading data from only one project, your Forseti service account 
+might have access only to that particular project. To get read access to all of 
+the projects under your organization, add the service account to the
+organization IAM policy with the 
+[required roles]({% link _docs/guides/best-practices.md %}#service-account-for-forseti-security).
+Your Organization Admin should be able to help you with that.
```
Comment (Tabrarian) [inline _faq/installation_and_deployment/org-data-access.md]: Maybe something a little more descriptive here, like "If you don't have permissions to change the organization policy, ask your Organization Admin for help."

File: _faq/security_implications/gsuite-key.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+title: Why does Forseti store the G Suite private key in the Compute Engine instnce?
+order: 1
+---
+{::options auto_ids="false" /}
+
+The 
+[Admin API](https://developers.google.com/admin-sdk/directory/v1/guides/delegation), 
+which performs the G Suite Groups data retrieval, uses methods from an OAuth library 
+which expect the private key to be local to where the code is running. 
+To minimize G Suite service account access, don't assign any IAM roles to it
```
Comment (Tabrarian) [inline _faq/security_implications/gsuite-key.md]: Cloud IAM

File: _faq/security_implications/gsuite-key.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+order: 1
+---
+{::options auto_ids="false" /}
+
+The 
+[Admin API](https://developers.google.com/admin-sdk/directory/v1/guides/delegation), 
+which performs the G Suite Groups data retrieval, uses methods from an OAuth library 
+which expect the private key to be local to where the code is running. 
+To minimize G Suite service account access, don't assign any IAM roles to it 
+and only grant the Groups/Group Members Read-Only scope in G Suite.
```
Comment (Tabrarian) [inline _faq/security_implications/gsuite-key.md]: This was a little hard to parse, maybe "... and limit Groups/Group Members to Read-Only scope in G Suite." ?

File: _faq/security_implications/read-data.md [hunk 11]
```diff
@@ -0,0 +1,15 @@
+---
+title: How is Forseti able to read data from all my projects?
+order: 2
+---
+{::options auto_ids="false" /}
+
+Forseti uses a service account which is granted roles on the organization 
+IAM policy. Because roles are hierarchical in GCP, any role that someone has 
+on the organization level IAM will also be inherited on a lower level, such as 
+at the folder or project level. For example, if you grant the "Browser" role
```
Comment (Tabrarian) [inline _faq/security_implications/read-data.md]: "... if someone has a Cloud IAM role on the organization level, the role is inherited by lower levels like the folder or project."

File: _faq/security_implications/read-data.md [hunk 12]
```diff
@@ -0,0 +1,15 @@
+---
+title: How is Forseti able to read data from all my projects?
+order: 2
+---
+{::options auto_ids="false" /}
+
+Forseti uses a service account which is granted roles on the organization 
+IAM policy. Because roles are hierarchical in GCP, any role that someone has
```
Comment (Tabrarian) [inline _faq/security_implications/read-data.md]: Cloud IAM

File: _faq/using_forseti/logs.md [hunk 13]
```diff
@@ -0,0 +1,13 @@
+---
+title: Where are the logs stored for Forseti?
+order: 3
+---
+{::options auto_ids="false" /}
+
+The installation log is stored in `/tmp/deployment.log` on the Forseti GCE
```
Comment (Tabrarian) [inline _faq/using_forseti/logs.md]: Compute Engine
